src/pages/Wishlist.jsx

```diff
@@ -1,7 +1,38 @@
+// import { CheckoutCard } from "../components";
+import { ProductCard } from "../components"
+import { useWish } from "../context/WishContext";
 const Wishlist = () => {
-    return(
-        <div className="Page-container">This is Wishlist</div>
-    )
-}
+  const { wishState } = useWish();
+  const { wishBasket } = wishState;
 
-export default Wishlist;
\ No newline at end of file
+  return (
+    <div className="Page-container">
+      <div className="section-heading flex-center p-1 font-lg">
+        My Wishlist({wishBasket.length})
+      </div>
+      
+        <div className="wish-grid">
+          {wishBasket.map((cartItem) => {
+            console.log(cartItem)
```

Comment:
You can destructure the item here and use its properties.
Remove console like logs

Reply:
thank you for pointing that out will make the changes.